src/experiment/trials/holdUpMarker.js

```diff
@@ -1,32 +1,13 @@
 import htmlButtonResponse from "@jspsych/plugin-html-button-response";
+import { LANGUAGE } from "../../config/main";
+import { h1 } from "../../lib/markup/tags";
+import { photodiodeGhostBox } from "../../lib/markup/photodiode";
 
-import { config, LANGUAGE } from "../../config/main";
-import { eventCodes } from "../../config/trigger";
-import { pdSpotEncode, photodiodeGhostBox } from "../../lib/markup/photodiode";
-import { div, h1, p } from "../../lib/markup/tags";
-
-// TODO @brown-ccv #330: Custom extension for EEG - this is initializeTriggerBox
-// TODO @brown-ccv #330: Need to ping the serial part - this isn't doing anything yet
-// TODO @brown-ccv: Prevent user from pressing continue until pdSpotEncode finishes (need to use jsPsych.pluginAPI.setTimeout)
 export const holdUpMarkerTrial = {
   type: htmlButtonResponse,
-  stimulus: function () {
-    const eventMarkerMarkup = h1(LANGUAGE.trials.eventMarker.connected, {
-      style: "color: green;",
-    });
-    return div(eventMarkerMarkup);
-  },
-  prompt: function () {
-    let holdUpMarkerPrompt = p(LANGUAGE.trials.holdUpMarker);
-
-    // Conditionally add the photodiodeGhostBox
-    if (config.USE_PHOTODIODE) holdUpMarkerPrompt += photodiodeGhostBox;
-
-    return holdUpMarkerPrompt;
-  },
+  stimulus:
+    h1(LANGUAGE.trials.eventMarker.connected, { style: "color: green;" }) +
```

Comment:
Let's remove this line for now. We don't actually check whether or not the event marker is connected so it doesn't make much sense to handle it here

```suggestion
```